applications/decoupledibpm/main.cpp

```diff
@@ -68,7 +68,7 @@ int main(int argc, char **argv)
     PetscReal dt = config["parameters"]["dt"].as<PetscReal>();
 
     // current time
-    PetscReal t = start * dt;
+    PetscReal t = config["parameters"]["tstart"].as<PetscReal>(start * dt);
```

Comment:
Given that now we can read `t` from HDF5 file, I suggest here we only declare `t` without initializing it, and initialize it in the following `if (start from beginning) else if (restart)`.